pandas/tests/groupby/methods/test_kurt.py

```diff
@@ -0,0 +1,27 @@
+import numpy as np
+
+import pandas as pd
+import pandas._testing as tm
+
+
+def test_groupby_kurt_equivalence():
+    # Test that that groupby kurt method (which uses libgroupby.group_kurt)
```

Comment:
Can you start tests with a comment referencing the GitHub issue.

    # GH#40139

Reply:
Added issue number.